agreements/project-contribution/asset-transfer-agreement.md

```diff
@@ -0,0 +1,216 @@
+
+### Controlling Agreement
+
+In the event of any conflict between the terms of this Transfer Agreement and the Fiscal Sponsorship Agreement, the terms of this Transfer Agreement shall control.
+
+### Amendments
+
+This Transfer Agreement may be amended only in writing signed by both parties.
+
+### Counterparts
+
+This Transfer Agreement may be executed in counterparts, each of which shall be deemed an original, but all of which together shall constitute one and the same instrument.
+
+By signing below, the parties agree to the terms and have executed this Asset Transfer Agreement as of the Effective Date.
+
+- **Assignor:** `__________________________________`  
+    `By:    ⎣_____________________________________________⎦`  
+    `Name:  ⎣_____________________________________________⎦`  
+    `Title: ⎣_____________________________________________⎦`  
+    `Date:  ⎣_____________________________________________⎦`  
+
+- **Sponsor:** *Commonhaus Foundation, Inc.*  
+    `By:    ⎣_____________________________________________⎦`  
+    `Name:  ⎣_____________________________________________⎦`  
+    `Title: ⎣_____________________________________________⎦`  
+    `Date:  ⎣_____________________________________________⎦`  
+
+## Schedules
+
+### SCHEDULE A: Registered Copyrights
+
+| Title | Country | Reg No |
+| ----- | ------- | ------ |
+| __    | __      | __     |
+
+&nbsp;
+
+### SCHEDULE B: Domain Names
+
+| Domain name | Registrar |
+| ----------- | --------- |
+| __          | __        |
+
+&nbsp;
+
+### SCHEDULE C: Hosting Accounts
+
+(List of web hosting accounts, document hosting platforms, build and testing servers, and cloud infrastructure)
+
+- __
+
+&nbsp;
+
+#### Source code repositories
+
+(List of individual repositories or organizations/groups of repositories)
+
+- __
+
+&nbsp;
+
+### SCHEDULE D: Social Media and Email Accounts
+
+(List of project-specific social media and email accounts)
+
+- __
+
+&nbsp;
+
+### SCHEDULE E: Trademarks
+
+#### Registered Trademarks
+
+| Trademark | Country | Appln Date | Appln No | Reg Date | Reg No |
+| --------- | ------- | ---------- | -------- | -------- | ------ |
+| __        | __      | __         | __       | __       | __     |
+
+&nbsp;
+
+#### Unregistered Trademarks
+
+(List; should be the same as what is in the Trademark List)
+
+- __
+
+&nbsp;
+
+## Exhibits
+
+### Exhibit A
+
+CONFIRMATORY TRADEMARK ASSIGNMENT
+
+WHEREAS, [Assignor] a [state or country of formation and type of entity] with an office located at [address] ("Assignor"), adopted and used in its business the trademarks as shown on the attached Schedule of Trademarks ("Marks"), incorporated by reference herein; and
+
+WHEREAS, Commonhaus Foundation, Inc., a Florida not for profit corporation with an office located at 6526 Old Brick Road, #120-175, Windermere, FL 34786 ("Assignee"), is acquiring the entire business of Assignor, including said Marks and the applications therefor or registrations thereof;
```

Comment:
> is acquiring the entire business of Assignor

This sounds intense. Is this legalese for just the trademarks, or literally the whole business?

Reply:
This exhibit was an example specifically for Trademarks. But if it makes you squirmy, it is probably the right time to start working on that inventory, as I think that will help draw the right boundaries around "the business" that you're transferring to the foundation.